compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -4815,6 +4815,11 @@ object Types {
   /** The type of an import clause tree */
   case class ImportType(expr: Tree) extends UncachedGroundType
 
+  /** Sentinal for typed export clauses */
+  @sharable case object ExportType extends CachedGroundType {
```

Comment:
Martin suggested using Unit instead of introducing a new type, can we do that?

Reply:
done